lib/tozny_model.dart

```diff
@@ -219,7 +219,7 @@ class AgentToken {
   @JsonKey(name: "token_type")
   String tokenType;
   @JsonKey(name: "expiry")
-  int expiry;
+  double expiry;
```

Comment:
After the changes to Dart was it confirmed that the Android plugin still runs? Nothing here looks like a breaking change, but just want to make sure

Reply:
good point. i'll verify before merging.

Reply:
Android plugin confirmed to work